client/lib/wpcom-undocumented/lib/undocumented.js

```diff
@@ -1492,6 +1492,18 @@ Undocumented.prototype.sitesNew = function( query, fn ) {
 	);
 };
 
+/**
+ * Launches a private site
+ *
+ * @param {string} - ID of the site to be launched
```

Comment:
Probably should clarify that this can also be a slug

Reply:
Good point. Implemented in [cc0060c](https://github.com/Automattic/wp-calypso/pull/30242/commits/cc0060c9055f4b32abdb90a0c063458932d9d960).